### ISSUE TITLE
v1.8.0-rc2 Testing: Call for help from the community 

### ISSUE THREAD

**Issue by jmarrec** [openstudiocoalition/OpenStudioApplication#735]
## Testing Instructions

A release candidate for the 1.8.0 release has been posted. The 1.8.0 is a major release. It updates the OpenStudio SDK from 3.7.0 to 3.8.0 (EnergyPlus from to 23.2.0 to 24.1.0).

The macOS installer is now codesigned!

**Are you willing to test it out to help us make a great product? This will only take a few minutes of your time, yet it is absolutely critical for us to be aware of bugs.**

Ready? You're awesome, thank you! 

Go and download the executable from the [v1.8.0-rc2 release](https://github.com/openstudiocoalition/OpenStudioApplication/releases/tag/v1.8.0-rc2). **Please take a look at the release notes as they showcase the new features and bug fixes, so you know what to look for**

Then try it out. You'll find a checklist below of things to try out, but please do feel free to go off-script, and if you can even try it out with your real world models that's even better.

**When you're done, please report back by posting on this issue, with the following template filled out.**

```
## Checklist

### Platform and version: <!--  eg: Ubuntu 20.04, macOS 10.15 x64, macOS 11.4 x64, macOS 12.3 arm64 (M1), Windows 10 20H2-->

* [ ] I could install the OpenStudioApp installer
* [ ] I could launch the OpenStudioApp.exe
* [ ] I can load File > Example > Example Model
* [ ] I can go to Geometry Preview and see the example model
* [ ] I can go to all tabs and see stuff loading
* [ ] I can create HVAC loops and assign a thermal zone
* [ ] I can search for Measures on the BCL
* [ ] I can put a measure in my workflow
* [ ] I can run a simulation
* [ ] I can see the simulation results (EnergyPlus Outputs, OpenStudio Results)
* [ ] I have tried enabling (and disabling) the Classic CLI
* [ ] I have tried creating and running a Python measure in the new CLI

## Problems found:

<!-- please describe -->
```

Thank you!

**Comment by jmarrec** [comment]
### Platform and version: Ubuntu 22.04 clean VM (.deb)

* [x] I could install the OpenStudioApp installer
* [x] I could launch the OpenStudioApp.exe
    * Getting "Could not find or download OpenStudio Results Measures"
* [x] I can load File > Example > Example Model
* [x] I can go to Geometry Preview and see the example model
* [x] I can go to all tabs and see stuff loading
* [x] I can create HVAC loops and assign a thermal zone
* [x] I can search for Measures on the BCL
    * I can actually download the "OpenStudio results" measure (I have to switch tabs before I see it in the "Library" on the Run Tab where I was before I clicked on" Find Measures")
    
![image](https://github.com/user-attachments/assets/696386a5-d070-400d-840c-43d9ec902cbb)

* [x] I can put a measure in my workflow
* [x] I can run a simulation
* [x] I can see the simulation results (EnergyPlus Outputs, OpenStudio Results)
* [x] I have tried enabling (and disabling) the Classic CLI
* [x] I have tried creating and running a Python measure in the new CLI

## Problems found:

![image](https://github.com/user-attachments/assets/5a676bd2-83ac-4757-9aad-6237f1318aa6)

---

The "Classic CLI" checkbox on the run tab appears unlinked to "Preferences > Use Classic CLI"

![image](https://github.com/user-attachments/assets/6777e3bb-662b-4b9d-98d2-a17b22bd7e61)

Similarly, I thought @macumber made the "MeasureManager" use always the C++ CLI, which is kinda weird because if on the Run Tab I used the "Use Classic CLI", then I can't create a Python Measures from the Run Tab.

**Comment by jmarrec** [comment]
### Platform and version: Windows 10 20H2 clean VM

* [x] I could install the OpenStudioApp installer
    * It is properly codesigned!  
* [ ] I could launch the OpenStudioApp.exe

![image](https://github.com/user-attachments/assets/bdc8ca07-43a8-4677-b17e-39f0b31d2bef)


* [ ] I can load File > Example > Example Model
* [ ] I can go to Geometry Preview and see the example model
* [ ] I can go to all tabs and see stuff loading
* [ ] I can create HVAC loops and assign a thermal zone
* [ ] I can search for Measures on the BCL
* [ ] I can put a measure in my workflow
* [ ] I can run a simulation
* [ ] I can see the simulation results (EnergyPlus Outputs, OpenStudio Results)
* [ ] I have tried enabling (and disabling) the Classic CLI
* [ ] I have tried creating and running a Python measure in the new CLI

## Problems found:

**Comment by manuvarkey** [comment]
Package not installable on Ubuntu 24.04 due to unmet dependencies.

```
sudo apt install ./OpenStudioApplication-1.8.0-rc2+cd925a1005-Ubuntu22.04.deb 
[sudo] password for manu: 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'openstudioapplication-1.8.0-rc2' instead of './OpenStudioApplication-1.8.0-rc2+cd925a1005-Ubuntu22.04.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 openstudioapplication-1.8.0-rc2 : Depends: libwxgtk3.0-0v5 (>= 3.0.0) but it is not installable or
                                            libwxgtk3.0-0 (>= 3.0.0) but it is not installable or
                                            libwxgtk3.0-gtk3-0v5 (>= 3.0.0) but it is not installable

```

**Comment by manuvarkey** [comment]
https://github.com/openstudiocoalition/OpenStudioApplication/blob/cd925a100551eff8bde8806d501655f216b391eb/CMakeLists.txt#L850

On Ubuntu 24.04, `libwxgtk3.2-1t64` seems to be the relevant library.

**Comment by jmarrec** [comment]
@manuvarkey icon not showing when under "Activities" (the launcher)?
Definitely works on 22.04
24.04 isn't officially supported (by openstudio SDK either for that matter) but we should do that.
Migjmht be this https://askubuntu.com/questions/1516383/missing-icons-in-dock-after-upgrade-to-ubuntu-24-04

3. What do you mean desktop font settings not respected? You mean like the text in osapp doesn't match whatever you picked as system font (eg Ubuntu Mono, courrier)? That's expected I guess?

**Comment by manuvarkey** [comment]
> @manuvarkey thanks for testing.
> 
>     2. Icon not showing when under "Activities" (the launcher)?
>        Definitely works on 22.04
>        24.04 isn't officially supported (by openstudio SDK either for that matter) but we should do that.
>        Migjmht be this https://askubuntu.com/questions/1516383/missing-icons-in-dock-after-upgrade-to-ubuntu-24-04
![Screenshot from 2024-09-04 00-14-42](https://github.com/user-attachments/assets/1dbb4d6b-815f-450f-a946-ccd41ca74cb1)

![Screenshot from 2024-09-04 00-14-02](https://github.com/user-attachments/assets/6d347143-8903-45ab-9e37-58b88cb3a154)


>     3. What do you mean desktop font settings not respected? You mean like the text in osapp doesn't match whatever you picked as system font (eg Ubuntu Mono, courrier)? That's expected I guess?

Yes. Other QT apps seems to work ok. Any way this is a minor detail.

![Screenshot from 2024-09-04 00-18-37](https://github.com/user-attachments/assets/4cf324af-d44a-40f3-a37e-d9e7805ff27e)

**Comment by jmarrec** [comment]
```
Graphics Feature Status
Canvas: Software only. Hardware acceleration disabled
Canvas out-of-process rasterization: Disabled
Direct Rendering Display Compositor: Disabled
Compositing: Software only. Hardware acceleration disabled
Multiple Raster Threads: Disabled
OpenGL: Disabled
Rasterization: Software only. Hardware acceleration disabled
Raw Draw: Disabled
Video Decode: Software only. Hardware acceleration disabled
Video Encode: Software only. Hardware acceleration disabled
Vulkan: Disabled
WebGL: Disabled
WebGL2: Disabled
WebGPU: Disabled
Driver Bug Workarounds
clear_uniforms_before_first_program_use
decode_encode_srgb_for_generatemipmap
disable_accelerated_vp8_decode
disable_accelerated_vp9_decode
disable_direct_composition
disable_dxgi_zero_copy_video
disable_vp_scaling
disable_vp_super_resolution
enable_webgl_timer_query_extensions
exit_on_context_lost
disabled_extension_GL_KHR_blend_equation_advanced
disabled_extension_GL_KHR_blend_equation_advanced_coherent
disabled_extension_GL_MESA_framebuffer_flip_y
Problems Detected
Gpu compositing has been disabled, either via blocklist, about:flags or the command line. The browser will fall back to software compositing and hardware acceleration will be unavailable.
Disabled Features: gpu_compositing
GPU process was unable to boot: GPU access is disabled through commandline switch --disable-gpu.
Disabled Features: all
GPU rasterization should only be enabled on NVIDIA and Intel and AMD RX-R2 GPUs with DX11+ or any GPU using ANGLE's GL backend.: [1047002](http://crbug.com/1047002)
Disabled Features: gpu_rasterization
Some drivers are unable to reset the D3D device in the GPU process sandbox
Applied Workarounds: exit_on_context_lost
Clear uniforms before first program use on all platforms: [124764](http://crbug.com/124764), [349137](http://crbug.com/349137)
Applied Workarounds: clear_uniforms_before_first_program_use
Direct composition flashes black initially on Win <10: [588588](http://crbug.com/588588)
Applied Workarounds: disable_direct_composition
Zero copy DXGI video hangs on shutdown on Win < 8.1: [621190](http://crbug.com/621190)
Applied Workarounds: disable_dxgi_zero_copy_video
Disable KHR_blend_equation_advanced until cc shaders are updated: [661715](http://crbug.com/661715)
Applied Workarounds: disable(GL_KHR_blend_equation_advanced), disable(GL_KHR_blend_equation_advanced_coherent)
Decode and Encode before generateMipmap for srgb format textures on Windows: [634519](http://crbug.com/634519)
Applied Workarounds: decode_encode_srgb_for_generatemipmap
VPx decoding isn't supported well before Windows 10 creators update.: [616318](http://crbug.com/616318), [667532](http://crbug.com/667532)
Applied Workarounds: disable_accelerated_vp8_decode, disable_accelerated_vp9_decode
Expose WebGL's disjoint_timer_query extensions on platforms with site isolation: [808744](http://crbug.com/808744), [870491](http://crbug.com/870491)
Applied Workarounds: enable_webgl_timer_query_extensions
Don't use video processor scaling on non-Intel, non-NVIDIA GPUs.: [993233](http://crbug.com/993233)
Applied Workarounds: disable_vp_scaling
Disable GL_MESA_framebuffer_flip_y for desktop GL: [964010](http://crbug.com/964010)
Applied Workarounds: disable(GL_MESA_framebuffer_flip_y)
VP8 decoding crashes before Windows 10 Fall Creators Update.: [1094840](http://crbug.com/1094840)
Applied Workarounds: disable_accelerated_vp8_decode
Don't use video processor super resolution on Intel Gen9 and older GPUs and non-Intel GPUs.: [1318380](http://crbug.com/1318380)
Applied Workarounds: disable_vp_super_resolution
Version Information
Data exported
2024-09-04T10:13:48.871Z
Chrome version
OpenStudioApp/1.8.0.0
Operating system
Windows NT 6.2.9200
Software rendering list URL
https://chromium.googlesource.com/chromium/src/+/d0c26951c9bf150bd78e0ead92495063dc7e6d24/gpu/config/software_rendering_list.json
Driver bug list URL
https://chromium.googlesource.com/chromium/src/+/d0c26951c9bf150bd78e0ead92495063dc7e6d24/gpu/config/gpu_driver_bug_list.json
ANGLE commit id
unknown hash
2D graphics backend
Skia/108 8c5e09f079a3e8eeb97532cf1cbc6e0934ce40f7
Command Line
"C:\openstudioapplication-1.8.0-rc2\bin\OpenStudioApp.exe" --ignore-gpu-blacklist --blink-settings=forceDarkModeEnabled=false --blink-settings=darkModeEnabled=false --remote-allow-origins=* --browser-subprocess-path="C:\openstudioapplication-1.8.0-rc2\bin\QtWebEngineProcess.exe" --application-name=OpenStudioApp --enable-threaded-compositing --disable-speech-api --disable-blink-features=EyeDropperAPI --enable-features=NetworkServiceInProcess2,TracingServiceInProcess --disable-features=ConsolidatedMovementXY,InstalledApp,EyeDropper,BackgroundFetch,WebOTP,WebPayments,WebUSB,PictureInPicture --use-gl=disabled --in-process-gpu --disable-gpu --gpu-preferences=UAAAAAAAAADoAAAIAAAAAAAAAAAAAAAAAABgAAAAAAAwAAAAAAAAAAAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAAEgAAAAAAAAASAAAAAAAAAAYAAAAAgAAABAAAAAAAAAAGAAAAAAAAAAQAAAAAAAAAAAAAAAOAAAAEAAAAAAAAAABAAAADgAAAAgAAAAAAAAACAAAAAAAAAA= --disable-direct-composition
Driver Information
Initialization time
0
In-process GPU
true
Passthrough Command Decoder
true
Sandboxed
false
GPU0
VENDOR= 0x0000, DEVICE=0x0000, LUID={0,0}
Optimus
false
AMD switchable
false
Desktop compositing
Aero Glass
Direct composition
false
Supports overlays
false
YUY2 overlay support
NONE
NV12 overlay support
NONE
BGRA8 overlay support
NONE
RGB10A2 overlay support
NONE
Driver D3D12 feature level
Not supported
Driver Vulkan API version
Not supported
GPU CUDA compute capability major version
0
Pixel shader version
Vertex shader version
Max. MSAA samples
Machine model name
Machine model version
GL_VENDOR
Disabled
GL_RENDERER
Disabled
GL_VERSION
Disabled
GL_EXTENSIONS
Disabled Extensions
GL_KHR_blend_equation_advanced GL_KHR_blend_equation_advanced_coherent GL_MESA_framebuffer_flip_y
Disabled WebGL Extensions
Window system binding vendor
Window system binding version
Window system binding extensions
Direct rendering version
unknown
Reset notification strategy
0x0000
GPU process crash count
0
gfx::BufferFormats supported for allocation and texturing
R_8: not supported, R_16: not supported, RG_88: not supported, RG_1616: not supported, BGR_565: not supported, RGBA_4444: not supported, RGBX_8888: not supported, RGBA_8888: not supported, BGRX_8888: not supported, BGRA_1010102: not supported, RGBA_1010102: not supported, BGRA_8888: not supported, RGBA_F16: not supported, YVU_420: not supported, YUV_420_BIPLANAR: not supported, P010: not supported
Compositor Information
Tile Update Mode
One-copy
Partial Raster
Enabled
GpuMemoryBuffers Status
R_8
Software only
R_16
Software only
RG_88
Software only
RG_1616
Software only
BGR_565
Software only
RGBA_4444
Software only
RGBX_8888
GPU_READ, SCANOUT
RGBA_8888
GPU_READ, SCANOUT
BGRX_8888
Software only
BGRA_1010102
Software only
RGBA_1010102
Software only
BGRA_8888
Software only
RGBA_F16
Software only
YVU_420
Software only
YUV_420_BIPLANAR
GPU_READ, SCANOUT
P010
Software only
Display(s) Information
Info
Display[2528732444] bounds=[0,0 1280x960], workarea=[0,0 1280x920], scale=1, rotation=0, panel_rotation=0 external.
Color space (sRGB/no-alpha)
{primaries:BT709, transfer:SRGB, matrix:RGB, range:FULL}
Buffer format (sRGB/no-alpha)
BGRX_8888
Color space (sRGB/alpha)
{primaries:BT709, transfer:SRGB, matrix:RGB, range:FULL}
Buffer format (sRGB/alpha)
BGRA_8888
Color space (WCG/no-alpha)
{primaries:BT709, transfer:SRGB, matrix:RGB, range:FULL}
Buffer format (WCG/no-alpha)
BGRX_8888
Color space (WCG/alpha)
{primaries:BT709, transfer:SRGB, matrix:RGB, range:FULL}
Buffer format (WCG/alpha)
BGRA_8888
Color space (HDR/no-alpha)
{primaries:BT709, transfer:SRGB, matrix:RGB, range:FULL}
Buffer format (HDR/no-alpha)
BGRX_8888
Color space (HDR/alpha)
{primaries:BT709, transfer:SRGB, matrix:RGB, range:FULL}
Buffer format (HDR/alpha)
BGRA_8888
Color volume
{name:'srgb', r:[0.6400, 0.3300], g:[0.3000, 0.6000], b:[0.1500, 0.3300], w:[0.3127, 0.3290]}
SDR white level in nits
203
HDR relative maximum luminance
1
Bits per color component
8
Bits per pixel
24
Refresh Rate in Hz
64
Video Acceleration Information
Decoding
Encoding
Vulkan Information
Device Performance Information
Total Physical Memory (Gb)
23
Total Disk Space (Gb)
0
Hardware Concurrency
4
System Commit Limit (Gb)
27
D3D11 Feature Level
Unknown
Has Discrete GPU
unknown
Software Rendering
No
Diagnostics
Driver Information for Hardware GPU
Initialization time
0
In-process GPU
true
Passthrough Command Decoder
true
Sandboxed
false
GPU0
VENDOR= 0x0000, DEVICE=0x0000, LUID={0,0}
Optimus
false
AMD switchable
false
Desktop compositing
Aero Glass
Direct composition
false
Supports overlays
false
YUY2 overlay support
NONE
NV12 overlay support
NONE
BGRA8 overlay support
NONE
RGB10A2 overlay support
NONE
Driver D3D12 feature level
Not supported
Driver Vulkan API version
Not supported
GPU CUDA compute capability major version
0
Pixel shader version
Vertex shader version
Max. MSAA samples
Machine model name
Machine model version
GL_VENDOR
Disabled
GL_RENDERER
Disabled
GL_VERSION
Disabled
GL_EXTENSIONS
Disabled Extensions
Disabled WebGL Extensions
Window system binding vendor
Window system binding version
Window system binding extensions
Direct rendering version
unknown
Reset notification strategy
0x0000
GPU process crash count
0
gfx::BufferFormats supported for allocation and texturing
R_8: not supported, R_16: not supported, RG_88: not supported, RG_1616: not supported, BGR_565: not supported, RGBA_4444: not supported, RGBX_8888: not supported, RGBA_8888: not supported, BGRX_8888: not supported, BGRA_1010102: not supported, RGBA_1010102: not supported, BGRA_8888: not supported, RGBA_F16: not supported, YVU_420: not supported, YUV_420_BIPLANAR: not supported, P010: not supported
Graphics Feature Status for Hardware GPU
Canvas: Software only. Hardware acceleration disabled
Canvas out-of-process rasterization: Disabled
Direct Rendering Display Compositor: Disabled
Compositing: Software only. Hardware acceleration disabled
Multiple Raster Threads: Disabled
OpenGL: Disabled
Rasterization: Software only. Hardware acceleration disabled
Raw Draw: Disabled
Video Decode: Software only. Hardware acceleration disabled
Video Encode: Software only. Hardware acceleration disabled
Vulkan: Disabled
WebGL: Disabled
WebGL2: Disabled
WebGPU: Disabled
Problems Detected for Hardware GPU
Gpu compositing has been disabled, either via blocklist, about:flags or the command line. The browser will fall back to software compositing and hardware acceleration will be unavailable.
Disabled Features: gpu_compositing
GPU process was unable to boot: GPU access is disabled through commandline switch --disable-gpu.
Disabled Features: all
Log Messages
```

**Comment by jmarrec** [comment]
* I just released 1.8.0-rc3, and will close this in favor of #743
